shell/dialog/ScaleRke1NodeDownDialog.vue

```diff
@@ -0,0 +1,117 @@
+<script>
```

Comment:
Other than the three cases that you mentioned above there's one more case when pointing to `cluster`:

```
const cluster = this.isRke2 ? this.resources[0].cluster : this.resources[0].provisioningCluster;
```

and `name` is actually `undefined` in the RKE1 `resources` object that we pass as a prop to the modal component.

My intention was to avoid adding more logic to the component specially since RKE1 could get deprecated in the future, but overall I don't have a strong opinion here so I combined them.